public/gl-fonts/README.md

```diff
@@ -0,0 +1 @@
+Generate me using https://github.com/GlobalFishingWatch/fonts
```

Comment:
There are actually only two typefaces. The big number of files generated is because for each font, there are as many files as ASCII character ranges. If we were to not use accented characters at all we could probably get away with just the 1st file for each font, but at the same time it doesn't hurt having them all, right?